charts/flyte-core/values.yaml

```diff
@@ -15,7 +15,7 @@ flyteadmin:
   replicaCount: 1
   image:
     # -- Docker image for Flyteadmin deployment
-    repository: cr.flyte.org/flyteorg/flyteadmin
+    repository: cr.flyte.org/flyteorg/flyteadmin # FLYTEADMIN_IMAGE
```

Comment:
Shouldn't these change to be
```suggestion
    repository: cr.flyte.org/flyteorg/flyteadmin-release # FLYTEADMIN_IMAGE
```
And the same for the others?

Reply:
No that is not required, We only prepare them for release.

Reply:
@EngHabu it's ready now